### PR TITLE
Add HTMLLint

### PR DESCRIPTION
**Pull request by pdehaan**
Ref: #56

### CODE REVIEW

File: views/home.hbs [hunk 1]
```diff
@@ -48,7 +48,7 @@
     <div class="reveal" id="subscribe-modal" data-reveal>
       <h3>Subscribe to Breach Alerts</h3>
       <p>Explanation of what the user will get if they subscribe to breach alerts.</p>
-      <a href="#" class="button sign-in-btn" onClick="doOauth();">Subscribe with Firefox Accounts</a>
+      <a href="#" class="button sign-in-btn" onclick="doOauth();">Subscribe with Firefox Accounts</a>
```
Comment (groovecoder) [inline views/home.hbs]: Is there a way to configure HTMLLint to disallow `on*` attributes? These are security no-no.

Comment (pdehaan) [reply views/home.hbs]: As close as I can tell, we'd need to use the [`attr-bans`](https://github.com/htmllint/htmllint/wiki/Options#attr-bans) rule, and explicitly specify any `on*` attribute we wanted to disallow. Not great, but I don't see any other easy config or RegExp in the docs [yet].

**Comment by pdehaan** [comment]
> We should add this to `.travis.yml` if we're going to use it.

❤️ No need. It's a `lint:*` task, which gets run automatically for each PR.
https://travis-ci.org/mozilla/blurts-server/jobs/346641658#L1668-L1672

File: views/home.hbs (hunk 1 above)
Comment (pdehaan) [reply views/home.hbs]: Oops, missed this...
Added `"onclick"` to the `attr-bans` rule... which now makes the CI fail.
Ideas on how to tweak the breakage? Or are we cool w/ merging a PR that fails the CI and we can submit a follow-up issue?

Comment (groovecoder) [reply views/home.hbs]: Let's not merge a CI fail. Are you comfortable moving the script behavior into a separate `.js` file yourself, or prefer someone else do it?

**Comment by groovecoder** [comment]
Superseded by https://github.com/mozilla/blurts-server/pull/66.

**Pull request closed without merging**